### PR TITLE
Update AUTHORS and LICENSE

### PR DESCRIPTION
**Pull request by patiencedaur**

### CODE REVIEW

**Comment by patiencedaur** [comment]
@NickVolynkin, is there anyone I missed among the authors?

File: AUTHORS [hunk 1]
```diff
@@ -3,55 +3,35 @@ many contributions from the community.
 
 Below is a list of people who contributed to the Tarantool documentation:
 
-Aleksandr Lyapunov
```
Comment (Totktonada) [inline AUTHORS]: I see that many real authors of the documentation were removed. Many past employees at least, including one that was in the documentation team.

The repository was extracted several years ago (seven or kinda) from the main repository. Some contributions are go from the main repository, but git history will not show them.

Also I would note that even if a developer does not technically create a commit, but wrote most of the content regarding a feature, he/she anyway the author and contributor.

File: AUTHORS [hunk 2]
```diff
@@ -3,55 +3,35 @@ many contributions from the community.
...
+Roman Gorev
 Roman Tsisyk
-Teodor Sigaev
-Timofey Khryukin
+Sergey Aleshin
+Sergey Tatus
+Sofia Yemelyanova
+Vadim Popov
 Valeria Beresneva
-Vassiliy Soshnikov
```
Comment (Totktonada) [inline AUTHORS]: Vasiliy writes his name differently: https://github.com/dedok.

File: AUTHORS (hunk 1 above)
Comment (Totktonada) [reply AUTHORS]: > <...> including one that was in the documentation team.

Ah, no. My mistake. I thought that Sofia was removed from the list.

Comment (patiencedaur) [reply AUTHORS]: Oh. My bad. I thought it was just copy-paste from the tarantool repo, updated manually, and had little to do with actual documentation authors. Will return the original authors to the list.

File: AUTHORS [hunk 3]
```diff
@@ -1,17 +1,37 @@
...
+Pavel Semyonov
+Peter Gulutzan
+Roman Gorev
+Roman Tsisyk
+Sergey Aleshin
+Sergey Tatus
+Sofia Yemelyanova
+Vadim Popov
+Valeria Beresneva
+Vladimir Izmalkov
```
Comment (NickVolynkin) [inline AUTHORS]: I've added back a few names which were in the original list

```suggestion
Albert Sverdlov
Aleksandr Lyapunov
Aleksey Demakov
Aleksey Mashanov
Alexander Turenko
Alexandra Mirzuitova
Alexandre Kalendarev
Andrey Drozdov
Anton Barabanov
Artem Morozov
Artur Barsegyan
Damien Lefortier
Dmitry E. Oboukhov
Dmitry Simonenko
Elena Shebunyaeva
Eugene Blikh
Eugene Leonovich
Eugene Osintsev
Eugene Shadrin
Evgeniy Osintsev
Georgy Kirichenko
Georgy Moiseev
Konstantin Knizhnik
Konstantin Nazarov
Konstantin Osipov
Konstantin Shulgin
Kseniia Antonova
Maria Strabykina
Marko Kevac
Mons Anderson
Natalia Ogoreltseva
Nick Volynkin
Nick Zavaritsky
Oleg Tsarev
Olga Turchanovskaya
Patience Daur
Pavel Cherenkov
Pavel Lapaev
Pavel Semyonov
Peter Gulutzan
Roman Antipin
Roman Gorev
Roman Tokarev
Roman Tsisyk
Sergey Aleshin
Sergey Tatus
Sofia Yemelyanova
Teodor Sigaev
Timofey Khryukin
Vadim Popov
Valeria Beresneva
Vasiliy Soshnikov
Veniamin Gvozdikov
Vladimir Izmalkov
Vladimir Rudnyh
Yuriy Nevinitsin
Yuriy Vostrikov

```

File: AUTHORS [hunk 4]
```diff
@@ -1,17 +1,63 @@
...
+Teodor Sigaev
+Timofey Khryukin
+Vadim Popov
+Valeria Beresneva
+Veniamin Gvozdikov
+Vasiliy Soshnikov
+Vladimir Izmalkov
+Vladimir Rudnyh
+Yuriy Nevinitsin
+Yuriy Vostrikov
```
Comment (NickVolynkin) [inline AUTHORS]:
```suggestion
Albert Sverdlov
Aleksandr Lyapunov
Aleksey Demakov
Aleksey Mashanov
Alexander Turenko
Alexandra Mirzuitova
Alexandre Kalendarev
Andrey Drozdov
Anton Barabanov
Artem Morozov
Artur Barsegyan
Damien Lefortier
Dmitry E. Oboukhov
Dmitry Simonenko
Elena Shebunyaeva
Eugene Blikh
Eugene Leonovich
Eugene Shadrin
Evgeniy Osintsev
Georgy Kirichenko
Georgy Moiseev
Konstantin Knizhnik
Konstantin Nazarov
Konstantin Osipov
Konstantin Shulgin
Kseniia Antonova
Maria Strabykina
Marko Kevac
Mons Anderson
Natalia Ogoreltseva
Nick Volynkin
Nick Zavaritsky
Oleg Tsarev
Olga Turchanovskaya
Patience Daur
Pavel Cherenkov
Pavel Lapaev
Pavel Semyonov
Peter Gulutzan
Roman Antipin
Roman Gorev
Roman Tokarev
Roman Tsisyk
Sergey Aleshin
Sergey Tatus
Sofia Yemelyanova
Teodor Sigaev
Timofey Khryukin
Vadim Popov
Valeria Beresneva
Vasiliy Soshnikov
Veniamin Gvozdikov
Vladimir Izmalkov
Vladimir Rudnyh
Yuriy Nevinitsin
Yuriy Vostrikov
```

**Review by Totktonada — APPROVED** [review]
The idea is okay for me. But I didn't do a fact check (compare lists and so on).

**Review by NickVolynkin — APPROVED** [review]

**Review by kyukhin — APPROVED** [review]

**Pull request merged**